### PR TITLE
Organizing setup related files.

### PR DESCRIPTION
**Pull request by mirons-google**

### CODE REVIEW

File: setup/build_protos.py [hunk 1]
```diff
@@ -21,7 +21,12 @@
 
 
 def is_grpc_service_dir(files):
-    """Returns true iff the directory hosts a gRPC service."""
+    """Returns true iff the directory hosts a gRPC service.
```
Comment (carise) [inline setup/build_protos.py]: newline after first line in pydoc

File: setup/build_protos.py [hunk 2]
```diff
@@ -21,7 +21,12 @@
...
+        Args:
+            files(list): A list of files from os.walk().
```
Comment (carise) [inline setup/build_protos.py]: newline after

File: setup/build_protos.py [hunk 3]
```diff
@@ -46,7 +51,10 @@ def clean():
 
 
 def make_proto_service(root):
-    """Generate a proto service from the definition file."""
+    """Generate a proto service from the definition file.
```
Comment (carise) [inline setup/build_protos.py]: newline

File: setup/build_protos.py [hunk 4]
```diff
@@ -46,7 +51,10 @@ def clean():
...
+        Args:
+            root(str): The root of the current working directory.t
```
Comment (carise) [inline setup/build_protos.py]: remove "t" at the end

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request closed without merging**